docs/src/pages/docs/installation/configuring.mdx

```diff
@@ -118,6 +118,13 @@ Make sure the pip package [`Authlib`](https://authlib.org/) is installed on the
 First, configure authorization in Superset `superset_config.py`.
 
 ```python
+from flask_appbuilder.security.manager import (
+    AUTH_OID,
+    AUTH_REMOTE_USER,
+    AUTH_DB,
+    AUTH_LDAP,
+    AUTH_OAUTH,
```

Comment:
nit: you can just import this one `AUTH_OAUTH`

Reply:
hi @dpgaspar thanks for your answer, actually this is exactly the point of my request, I've spent an entire day trying to make the SSO with Google work and all I had to do was add the rest of the libraries in order to make it work, so as is, the current documentation will fail in step to step implementation

Reply:
It doesn't give a crash log, it just doesn't work, I was trying the OAUTH with google, and didn't authenticate until I added the rest of the dependencies. I've spent about 10 days on that